src/app/page/categoryInfo/deck/deck.component.html

```diff
@@ -17,13 +17,14 @@
                           [rating]="deck.rating"></star-rating-comp>
       </td>
       <td class="alignCenter">
-        <div (click)="addDeckToFolder(deck.deckId)" *ngIf="!deck.status; else other_content">
-          <div *ngIf="isAuthorized()" class="button addDeckToUserButtonActive" title="Add deck to cabinet"></div>
+        <div *ngIf="!deck.status">
+          <div (click)="addDeckToFolder(deck.deckId)" *ngIf="isAuthorized()" class="button addDeckToUserButtonActive"
+               title="Add deck to cabinet"></div>
         </div>
-        <ng-template #other_content>
-          <div *ngIf="isAuthorized()" class="button addDeckToUserButtonChecked"
+        <div *ngIf="deck.status">
+          <div (click)="addDeckToFolder(deck.deckId)" *ngIf="isAuthorized()" class="button addDeckToUserButtonChecked"
```

Comment:
Is it possible to use here ng else statement?

Reply:
I don't think so. It was done with ng else statement but the error was shown, so I replaced it with just *ngIf statement.